spec/models/event_spec.rb

```diff
@@ -1,8 +1,168 @@
 require 'spec_helper'
 
 describe Event do
+  subject { create(:event) }
+  let(:conference) { create(:conference) }
+  let(:event) { create(:event, program: conference.program) }
+  let(:user) { create(:user) }
 
-  describe 'abstract_word_count' do
+  describe 'association' do
+    it 'should belong to program' do
+      should belong_to :program
+    end
+
+    it 'should belong to event_type' do
+      should belong_to :event_type
+    end
+  end
+
+  describe 'validation' do
+    it 'has a valid factory' do
+      expect(build(:event)).to be_valid
+    end
+
+    it 'is not valid without a title' do
+      should validate_presence_of(:title)
+    end
+
+    it 'is not valid without an abstract' do
+      should validate_presence_of(:abstract)
+    end
+
+    it 'is not valid without a program' do
+      should validate_presence_of(:program)
+    end
+
+    it 'is not valid without an event_type' do
+      should validate_presence_of(:event_type)
+    end
+
+    describe '#abstract_limit' do
+      before :each do
+        event.event_type.maximum_abstract_length = 2
+        event.event_type.minimum_abstract_length = 2
+      end
+
+      context 'is invalid' do
+
+        it 'when abstract is too long' do
+          event.abstract = 'Test abstract here'
+          expect(event.valid?).to eq false
+          expect(event.errors[:abstract]).to eq ['cannot have more than 2 words']
+        end
+
+        it 'when abstract is too short' do
+          event.abstract = 'Test'
+          expect(event.valid?).to eq false
+          expect(event.errors[:abstract]).to eq ['cannot have less than 2 words']
+        end
+      end
+
+      context 'is valid' do
+        it 'when abstract length is within limits' do
+          event.abstract = 'Test abstract'
+          expect(event.valid?).to eq true
+          expect(event.errors.size).to eq 0
+        end
+      end
+    end
+
+    describe '#before_end_of_conference' do
+      context 'is invalid' do
+        before :each do
```

Comment:
Not sure if it really necessary to have this in a before block if you only have one test, but no strong preference here.
(same for the next test)